### PR TITLE
Docs page

### PR DESCRIPTION
**Pull request by harsh788**
**What does this PR do?**
This PR adds a new page in the website, which helps users to learn on how to contribute to the project.

**Related Issue(s)**
https://github.com/Vijaykv5/SnapGrid/issues/160#issue-1944010660

**Changes**
This PR adds a new page in the website, which helps users to learn on how to contribute to the project. A new npm package, `react-code-blocks` has been added in the project.

**Screenshots or GIFs (if applicable)**
If your PR includes visual changes, provide screenshots or GIFs to demonstrate the changes.
Screenshot from 2023-10-23 02-59-13
Screenshot from 2023-10-23 02-59-24
Screenshot from 2023-10-23 02-59-37

**Checklist**
- [x] I have read and followed the [contributing guidelines](../CONTRIBUTING.md).
- [x] My code follows the project's coding style and conventions.
- [ ] I have added unit tests (if applicable) to ensure the code works as expected.
- [ ] I have updated the project documentation (if necessary).

**Additional Comments**

**By submitting this PR, I confirm that:**
- [x] I have reviewed my code and believe it is ready for merging.
- [x] I understand that this PR may be subject to review and changes.
- [x] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

**Comment by Vijaykv5** [comment]
@harsh788 Please Fix those Changes I suggested

**Comment by harsh788** [comment]
Can you please guide me where are the changes mentioned. I can't seem to find them.

**Comment by Vijaykv5** [comment]
> Can you please guide me where are the changes mentioned. I can't seem to find them.

I mentioned it on above comment

**Comment by Vijaykv5** [comment]
Look at the review @harsh788

**Comment by harsh788** [comment]
I am not sure if the review is visible to me. I checked everywhere. It also says `No reviews` in the sidetab

**Comment by Vijaykv5** [comment]
I hope you can see it now @harsh788

File: src/Pages/DocsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            <span className='text-[#6F4FF2] font-bold'>Step 6:</span> Install the dependencies using npm install. Start the local server using npm start. 
+            <CopyBlock 
+              text={`npm install 
+npm start`} 
+              showLineNumbers={false} 
+              language='bash'
+              codeBlock 
+              theme={shadesOfPurple}
+            />
+          </div>
```
[the remark this answers is not in the document]
Comment (harsh788) [reply src/Pages/DocsPage.jsx:75]: Should I copy paste the command `echo "REACT_APP_UNSPLASH_API_KEY=your-unsplash-api-key" > .env` as it is in the code block?

**Comment by harsh788** [comment]
Screenshot from 2023-10-24 15-28-14
I have made required changes @Vijaykv5.

**Comment by Vijaykv5** [comment]
Thanks for your contribution @harsh788  Your work is greatly appreciated.
If you find this project helpful, please consider giving it a star ⭐️.
It helps to support the project and let others discover it too!

**Pull request merged**